README.md

```diff
@@ -16,7 +16,7 @@
 
 ## Link to the live page
 
-> Feel free to check the []().
+> Feel free to check the [](https://tapishkhari.github.io/FirstDevProject/).
```

Comment:
- [ ] Kindly add a title `[live demo](https://tapishkhari.github.io/FirstDevProject/)` so your link is visible and clickable [live demo](https://tapishkhari.github.io/FirstDevProject/) 👍🏿

Reply:
@prolajumokeoni : I have made the changes as per the requirement.